src/bluetooth/DevBleManager.js

```diff
@@ -1,3 +1,16 @@
+const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+const randomMacs = [
+  'AA:BB:CC:DD:EE:FF',
+  '12:34:56:78:09:10',
+  'AB:CD:EF:GH:IJ:KL',
+  'TH:IS:IS:TO:TE:SA',
+  'RE:AL:MA:CO:MG:AA',
+  'HE:HE:WO:ND:ER:IF',
+  'AN:YO:NE:WI:LL:NO',
```

Comment:
Added this to have multiple options when scanning

Reply:
🤣

Reply:
Is it missing a 'T'?
`THIS IS TO TES[T] A REAL MAC OMG AA HE HE WONDER IF ANYONE WILL NOTICE THIS JOKE`

Reply:
woah totes crazy!